pint/random_models.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
```

Comment:
Try running flake8 (pip install flake8 if you don't have it) over this file. It checks for python style issues we should follow as well as we can for new code. (Converting existing code is a bigger project.)